static/sass/style.scss

```diff
@@ -0,0 +1,6 @@
+@import "partials/base";
+@import "partials/content";
+@import "partials/footer";
+@import "partials/global";
```

Comment:
Could you move global to be second, after base?

Reply:
Done